src/main.tsx

```diff
@@ -3,11 +3,14 @@ import { createRoot } from "react-dom/client";
 import "./styles/main.css";
 import App from "./App";
 import { AppProvider } from "./contexts/AppContext";
+import { BrowserRouter } from "react-router-dom";
 
 createRoot(document.getElementById("root")!).render(
   <StrictMode>
-    <AppProvider>
-      <App />
-    </AppProvider>
+    <BrowserRouter>
```

Comment:
Could you move the BrowserRouter to `App.tsx` just wrapping the routes ?

Reply:
Moved it, however the header needs to be wrapped as I use the route hook from the searchbar.